chatwoot/chatwoot
# fix: Fetch conversationAttributes after contact reset

- Fetch conversation attributes after the contact is reset.

Fixes https://github.com/chatwoot/chatwoot/issues/3825


**To test:** 

- Load /widget_tests with setUser=true flag
- Create a conversation and resolve it (Make sure no messages are allowed after resolution). You would see create new conversation button.
- Go to the browser console. Remove all the cookies. 
- Reload the page with setUser=true
- The last conversation should be loaded without any issues. Before this change, the start new conversation button would not be shown instead it would show a message input.
